server/database/db_build.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+const dbConnection = require('./dbConnection');
+
+const makeEmptyTables = fs.readFileSync(`${__dirname}/build.sql`, 'utf-8');
+
+const runDbBuild = () => dbConnection.query(makeEmptyTables, (error) => {
+  if (error) {
+    console.log(error);
```

Comment:
If you want to keep your console logs here, you should disable eslint warnings. (check the docs for how to do it)